doc/manual/packages.rst

```diff
@@ -570,11 +570,19 @@ going to have to do a *force push*:
   indicates that this should replace the ``fixbar`` branch found at
   ``myfork``.
 
-
 Creating a new Package
 ----------------------
 
-Guidelines for Naming a Package
+REQUIRE speaks for itself
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should have a ``REQUIRE`` file in your package repo, with a bare minimum directive of what Julia version to include.
+When Julia version ``0.x`` is released, use ``julia 0.x`` in your ``REQUIRE``, not ``julia 0.x-``. You may consider using
```

Comment:
I would explain what `0.x-` means before saying not to use it. In this version, my reaction was: "why do you tell me about this if I'm not supposed to use it." :-) So, first say what to do to depend on a released version, and then what to (possibly) do if you depend on a development version.